djconnectwise/sync.py

```diff
@@ -211,6 +211,8 @@ def persist_page(self, records, results):
                     results.updated_count += 1
                 else:
                     results.skipped_count += 1
+            except IntegrityError as e:
```

Comment:
Since you are handling the exceptions the same way just group them together with `except (IntegrityError, InvalidObjectException) as e:`

Reply:
Thanks for catching this. Fixed.